packages/deployment/app/cloudformation.template.yml

```diff
@@ -50,60 +50,26 @@ Resources:
             - {{../es.name}}Domain
             - DomainEndpoint
       {{/if}}
-        KinesisInboundEventLoggerLambdaFunction:
-          Fn::GetAtt:
-            - KinesisInboundEventLoggerLambdaFunction
-            - Arn
-        KinesisOutboundEventLoggerLambdaFunction:
-          Fn::GetAtt:
-            - KinesisOutboundEventLoggerLambdaFunction
-            - Arn
         log2elasticsearchLambdaFunction:
           Fn::GetAtt:
             - log2elasticsearchLambdaFunction
             - Arn
-        ScheduleSFLambdaFunction:
-          Fn::GetAtt:
-            - ScheduleSFLambdaFunction
-            - Arn
-        dbIndexerLambdaFunction:
-          Fn::GetAtt:
-            - dbIndexerLambdaFunction
-            - Arn
-        messageConsumerLambdaFunction:
-          Fn::GetAtt:
-            - messageConsumerLambdaFunction
-            - Arn
-        CreateReconciliationReportLambdaFunction:
-          Fn::GetAtt:
-            - CreateReconciliationReportLambdaFunction
-            - Arn
-        BulkDeleteLambdaFunction:
-          Fn::GetAtt:
-            - BulkDeleteLambdaFunction
-            - Arn
+        kinesisFallbackSns:
+          Ref: kinesisFallbackSns
         EcsCluster:
           Ref: CumulusECSCluster
         AsyncOperationTaskDefinition:
           Ref: AsyncOperationTaskDefinition
-      {{#each ../apis}}
-        {{name}}RestApi:
-          Ref: {{name}}RestApi
-        {{name}}RestApiRootId:
+        {{#ifEquals @key "CumulusApiBackend"}}
```

Comment:
The backend API stack depends on the distribution API stack, but not vice versa. The backend API stack needs a reference to `distributionRestApi` because some backend API endpoints (granules) need the `DISTRIBUTION_ENDPOINT` env var, which is the URL to the distribution API built using the `distributionRestApi` ref